scripts/src/javatests/org/oppia/android/scripts/testfile/TestFileCheckTest.kt

```diff
@@ -0,0 +1,94 @@
+package org.oppia.android.scripts.testfile
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.oppia.android.scripts.common.TEST_FILE_CHECK_FAILED_OUTPUT_INDICATOR
+import org.oppia.android.scripts.common.TEST_FILE_CHECK_PASSED_OUTPUT_INDICATOR
+import org.oppia.android.testing.assertThrows
+import java.io.ByteArrayOutputStream
+import java.io.PrintStream
+
+/** Tests for [TestFileCheck]. */
+class TestFileCheckTest {
```

Comment:
We ought to have at least 1 test for exemptions (though there are clearly too many to test them all, and we don't necessarily want to test all of them, anyway).

Reply:
Added.